orc8r/cloud/go/obsidian/swagger/handlers/handlers_test.go

```diff
@@ -11,13 +11,13 @@
  * limitations under the License.
  */
 
-package handlers_test
+package handlers
```

Comment:
We want to use separate test pkg where at all possible. If you need to export `getCombinedSpecHandler` and similar, that's fine

I may have given conflicting PR advice about exporting -- but in this case, exporting those identifiers is better than having an in-pkg test file

Reply:
Got it, that's what I initially thought but I changed it because of the previous round of revision.